main/src/ca/uwaterloo/flix/language/phase/Namer.scala

```diff
@@ -370,65 +370,54 @@ object Namer {
     * Performs naming on the given enum case.
     */
   private def visitCase(case0: DesugaredAst.Case, enumSym: Symbol.EnumSym)(implicit flix: Flix, sctx: SharedContext): Validation[NamedAst.Declaration.Case, NameError] = case0 match {
-    case DesugaredAst.Case(ident, tpe0, loc) =>
-      mapN(visitType(tpe0)) {
-        case tpe =>
-          val caseSym = Symbol.mkCaseSym(enumSym, ident)
-          NamedAst.Declaration.Case(caseSym, tpe, loc)
-      }
+    case DesugaredAst.Case(ident, tpe, loc) =>
+      val t = visitType(tpe)
```

Comment:
I guess you don't like the `tpe0` convention? :)

Reply:
No not when it's not necessary :)